tsl/test/expected/continuous_aggs_policy.out

```diff
@@ -0,0 +1,97 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- initialize the bgw mock state to prevent the materialization workers FROM running
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+--TEST1 ---
+--basic test with count
+CREATE TABLE int_tab (a integer, b integer, c integer);
+SELECT TABLE_name FROM CREATE_hyperTABLE('int_tab', 'a', chunk_time_interval=> 10);
+NOTICE:  adding not-null constraint to column "a"
+ table_name 
+------------
+ int_tab
+(1 row)
+
+INSERT INTO int_tab VALUES( 3 , 16 , 20);
+INSERT INTO int_tab VALUES( 1 , 10 , 20);
+INSERT INTO int_tab VALUES( 1 , 11 , 20);
+INSERT INTO int_tab VALUES( 1 , 12 , 20);
+INSERT INTO int_tab VALUES( 1 , 13 , 20);
+INSERT INTO int_tab VALUES( 1 , 14 , 20);
+INSERT INTO int_tab VALUES( 2 , 14 , 20);
+INSERT INTO int_tab VALUES( 2 , 15 , 20);
+INSERT INTO int_tab VALUES( 2 , 16 , 20);
+CREATE OR REPLACE FUNCTION integer_now_int_tab() returns int LANGUAGE SQL STABLE as $$ SELECT coalesce(max(a), 0) FROM int_tab $$;
+SELECT set_integer_now_func('int_tab', 'integer_now_int_tab');
+ set_integer_now_func 
+----------------------
+ 
+(1 row)
+
+CREATE OR REPLACE VIEW mat_m1( a, countb )
+WITH (timescaledb.continuous, timescaledb.materialized_only=true)
+as
+SELECT a, count(b)
+FROM int_tab
+GROUP BY time_bucket(1, a), a;
+NOTICE:  adding index _materialized_hypertable_2_a_time_partition_col_idx ON _timescaledb_internal._materialized_hypertable_2 USING BTREE(a, time_partition_col)
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DELETE FROM _timescaledb_config.bgw_job WHERE TRUE;
+SET ROLE :ROLE_DEFAULT_PERM_USER;
+SELECT count(*) FROM _timescaledb_config.bgw_job;
+ count 
+-------
+     0
+(1 row)
+
+\set ON_ERROR_STOP 0
+SELECT add_refresh_continuous_aggregate_policy('int_tab', '1 day'::interval, 10 , '1 h'::interval); 
```

Comment:
Is it possible to avoid having to cast arguments here? I am thinking of something similar to what I did for `refresh_continuous_aggregate` where we use the time-type to infer the argument type if the type is `UNKNOWNOID`.